cohortextractor/__init__.py

```diff
@@ -4,7 +4,6 @@
 from .measure import Measure
 from .query_language import categorise, table
 
-
```

Comment:
This is a bit odd, isort is configured with `lines_after_imports = 2`, but it passed in CI so I don't think there's anything for you to do here, just flagging it!

Reply:
Have you seen that I've changed the isort config to be compatible with black?  (They were fighting.)